src/js/EpubReader.js

```diff
@@ -1064,9 +1064,11 @@ BookmarkData){
         var $appContainer = $('#app-container');
         $appContainer.empty();
         $appContainer.append(ReaderBody({strings: Strings, dialogs: Dialogs, keyboard: Keyboard}));
-        $('nav').empty();
-        $('nav').attr("aria-label", Strings.i18n_toolbar);
-        $('nav').append(ReaderNavbar({strings: Strings, dialogs: Dialogs, keyboard: Keyboard}));
+
```

Comment:
this nav selector is overly broad, and in other places #app-navbar is used, so nav's without #app-navbar don't even work.